src/applications/vaos/services/slot/index.unit.spec.jsx

```diff
@@ -33,25 +33,23 @@ describe('VAOS Services: Slot ', () => {
       const data = await getSlots({
         siteId: '983',
         clinicId: '983_308',
-        startDate: startDate.format('YYYY-MM-DD'),
-        endDate: endDate.format('YYYY-MM-DD'),
+        startDate: format(startDate, 'yyyy-MM-dd'),
+        endDate: format(endDate, 'yyyy-MM-dd'),
       });
 
       expect(global.fetch.firstCall.args[0]).to.contain(
-        `/vaos/v2/locations/983/clinics/308/slots?start=${moment(startDate)
-          .startOf('day')
-          .format('YYYY-MM-DDTHH:mm:ssZ')}&end=${moment(endDate)
-          .startOf('day')
-          .format('YYYY-MM-DDTHH:mm:ssZ')}`,
+        `/vaos/v2/locations/983/clinics/308/slots?start=${formatISO(
+          startOfDay(startDate),
+        )}&end=${formatISO(startOfDay(endDate))}`,
```

Comment:
I think we're jumping the gun a little here since we aren't currently formatting it in ISO, this will be the case after completing https://github.com/department-of-veterans-affairs/va.gov-team/issues/110237. Currently the format includes the timezones: `[2025-06-23T00:00:00+00:00](https://dev-api.va.gov/vaos/v2/locations/983/clinics/308/slots?start=2025-05-24T00:00:00+00:00&end=2025-06-23T00:00:00+00:00)`

Reply:
I think the code below is formatting in ISO format which hasn't been touch in a while.

```
export async function getSlots({ siteId, clinicId, startDate, endDate }) {
  try {
    console.log(startDate, moment(startDate).format())
    const data = await getAvailableV2Slots(
      siteId,
      clinicId.split('_')[1],
      moment(startDate).format(),
      moment(endDate).format(),
    );
    return transformV2Slots(data || []);
  } catch (e) {
    if (e.errors) {
      throw mapToFHIRErrors(e.errors);
    }

    throw e;
  }
}
```

Reply:
Right, I'm just suggesting we should the [ISODateTimeUTC format](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/constants.js#L504) instead of `formatISO` which is adding the timezone offset information. I believe that's why the test is failing in https://github.com/department-of-veterans-affairs/vets-website/actions/runs/15211277172/job/42785948822?pr=36672